Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift

```diff
@@ -0,0 +1,551 @@
+import CTensorFlow
+
+/// The `TF_Tensor *` type.
+typealias CTFTensor = OpaquePointer
+
+public class LazyTensor : _AnyTensorHandle {
+    enum Handle {
+        /// Bool indicates if this concrete TFETensorhandle was a result of
+        /// materialization.
+       case conc(TFETensorHandle, Bool)
+        /// Bool indicates whether this is held by a swift variable,
```

Comment:
```suggestion
        /// Bool indicates whether this is held by a Swift variable,
```

Also, I'm not sure I understand what "held by a Swift variable" means. Could you enlighten?

Reply:
This flag is just used to mark live tensors so that they can also be materialized whenever the materialization of any other tensor is triggered. I have updated the comment accordingly.